cdap-api/src/main/java/co/cask/cdap/api/artifact/ArtifactId.java

```diff
@@ -49,11 +49,7 @@ public ArtifactScope getScope() {
 
   @Override
   public String toString() {
-    return "ArtifactId{" +
-      "name='" + name + '\'' +
-      ", version=" + version +
-      ", scope='" + scope + '\'' +
-      '}';
+    return String.format("%s-%s-%s", scope, name, version);
```

Comment:
I think it's better not to localize based on `toString`. You can always construct the file name explicitly (like "name-version-scope.jar").